app/pages/profile/comment-link.cjsx

```diff
@@ -26,7 +26,7 @@ module?.exports = React.createClass
       @getCommentHREF(nextProps.comment)
 
   getCommentHREF: (comment) ->
-    @setState({href: '#'})
+    @setState({href: ''})
```

Comment:
This isn't necessary is it? Just change: https://github.com/zooniverse/Panoptes-Front-End/pull/1511/files#diff-28e40cf08cbf41290f440b8f9985060bR19